CvGameCoreDLL_Expansion2/CvBuilderTaskingAI.cpp

```diff
@@ -2639,7 +2639,8 @@ int CvBuilderTaskingAI::ScorePlotBuild(CvPlot* pPlot, ImprovementTypes eImprovem
 
 	// If we are performing a culture bomb, find which city will be owning the plot
 	bool bIsCultureBomb = pkImprovementInfo ? pkImprovementInfo->GetCultureBombRadius() > 0 : false;
-	if (bIsCultureBomb && !pOwningCity)
+	bool bIsTileClaim = pkImprovementInfo ? pkImprovementInfo->IsNewOwner() : false;
+	if ((bIsCultureBomb || bIsTileClaim) && !pOwningCity)
```

Comment:
Is `pkImprovementInfo` double check really required?
```suggestion
  bool bIsCultureBomb = false;
  bool bIsTileClaim = false;
  
  if (pkImprovementInfo)
  {
    bIsCultureBomb = pkImprovementInfo->GetCultureBombRadius() > 0;
    bIsTileClaim = pkImprovementInfo->IsNewOwner()
  }
	if (!pOwningCity && (bIsCultureBomb || bIsTileClaim))
```

Reply:
pkImprovementInfo null check is done quite a few times in this function, could be optimized but don't think it's a very heavy computation :)